src/style.css

```diff
@@ -0,0 +1,133 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+:root {
+  ---blue: #00acdc;
+}
+
+html {
+  font-family: "Roboto";
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 0;
+  box-sizing: border-box;
```

Comment:
move box-sizing to * selector
it will be apply this style for all selectors and help to fix tests for button